.github/workflows/partial-tests.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         run: pip install --upgrade setuptools wheel
 
       - name: Install pipenv
-        run: pip install --upgrade pipenv
+        run: pip install pipenv --upgrade
```

Comment:
Does the order really matter ?

Reply:
it does not but I changed it for consistency